devops/nightly.yml

```diff
@@ -10,7 +10,6 @@ schedules:
   branches:
     include:
       - master
-      - dev
```

Comment:
I can take care of these bits in a separate PR. I need to delete the 'schedule test' YAML anyway

Reply:
Sure, whatever works. It's not critical, I just stumbled over it and there's no harm in getting rid of it. Whoever goes first...

Reply:
I've already done these :-)